### DISCUSSION TITLE
Brushes ideas; Accessibility and expansions

### DISCUSSION THREAD

**Discussion by Landy005** [Mojang/minecraft-editor#127] [Feedback]
Since #10 references to new brushes and custom brushes I thought of some ideas;

-Brush history like photoshop including the settings. (The latter is because I'm guessing these standard brushes will have alot of customisation sliders)

-favourite brushes saved, including the settings (masks, mirrored, axis, materials, scale or stretching etc etc).

-Randomise settings per click. Being able to check a box per setting wether per time used it would randomise that slider setting. Perhaps even have a maximum or minimum slider in where it's allowed to randomise from.
Ex: Imagine a sphere with a radius slider. Per placement it would randomise the radius' value between 2 predefined values. That way we could get variation quickly without having too contrasting results that a full ranged random setting would bring.

-Another way of brushing terrain is height maps. How gobrush (plugin) and other tools like worldpainter (software) uses alpha channel height maps like perlin noise or more complex even real life height maps we can download online. 

My all-time favourite brush on java is arceon's (plugin) boulder brush, it mimics rocks very well and is the easiest way so far to get realistic terrain for builders without too much manual sculpting the shape. Which also brings me to my last thought;

-Sculpting brushes; things like pull and push, smoothing and roughening, lifting and flattening, which are often used to make organics or custom terrain (check out meta edits & meta balls plugins for amazing erode like operations on Minecraft terrain)

**Comment by minecraftchloe** [comment]
cc @rpoerschke and @v-ToKeen for brush option expansion ideas!
